Source/Renderer/CubeMap.js

```diff
@@ -121,25 +120,30 @@ define([
         gl.activeTexture(gl.TEXTURE0);
         gl.bindTexture(textureTarget, texture);
 
-        function createFace(target, sourceFace) {
+        function createFace(target, sourceFace, preMultiplyAlpha, flipY) {
+            // TODO: gl.pixelStorei(gl._UNPACK_ALIGNMENT, 4);
+            gl.pixelStorei(gl.UNPACK_PREMULTIPLY_ALPHA_WEBGL, false);
```

Comment:
Are there duplicate `pixelStorei` calls if `sourceFace.arrayBufferView` below is not defined?  If so, can we move these to the first `if` block?